exhale/graph.py

```diff
@@ -564,6 +565,19 @@ def typeSort(self):
         for c in self.children:
             c.typeSort()
 
+    def inPageHierarchy(self):
+        '''
+        Whether or not this node should be included in the page view hierarchy.  Helper
+        method for :func:`~exhale.graph.ExhaleNode.toHierarchy`.  Sets the member
+        variable ``self.in_page_hierarchy`` to True if appropriate.
+
+        :Return (bool):
+            True if this node should be included in the page view --- if it is a
+            node of kind ``page``. Returns False otherwise.
+        '''
+        self.in_page_hierarchy = (self.kind == "page")
```

Comment:
```diff
- self.in_page_hierarchy = (self.kind == "page")
+ self.in_page_hierarchy = self.kind == "page"
```

parens not needed.  from what I'm gathering, pages don't have any hierarchy?  seems legit